### DISCUSSION TITLE
🔗 🚀 Feature Request: Please support editing elementor on tablet (Duplicate of #23528)

### DISCUSSION THREAD

**Discussion by 9togethers** [elementor/elementor#21240] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I think that nowadays so many people are working on a tablet and it will be so convenient to adjust a things or testing a new idea while we are not in front of the computer.

### Describe the solution you'd like

The problem is now we can't edit for create new page with elementor elements on tablet both Android and ios. Please make it works

**Comment by nicholaszein** [comment]
Hello there, @9togethers!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #23528

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

Kind regards
